teachopencadd/talktorials/T032_compound_activity_proteochemometrics/talktorial.ipynb

```diff
@@ -0,0 +1,1923 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #27.        target_dataset = target_dataset.drop(columns=['SMILES', 'accession'])
</pre><p>A basic question here - why is it necessary to remove the identifiers here?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/teachopencadd/pull/278/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='73'/>

Reply:
<p>So the dataset for modelling does not include labels. We could keep them and select just the target variable and features for model fitting, but since we select them by position it is easier this way. It is also a good reminder that the modeling part cannot include labels. </p>